server/models/Request.js

```diff
@@ -0,0 +1,36 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema
+const ObjectId = Schema.Types.ObjectId
+
+const requestSchema = new mongoose.Schema({
+  ownerId: {
+    type: ObjectId,
+    required: true,
+  },
+  sitterId: {
+    type: ObjectId,
+    required: true,
+  },
+  startDate: {
+    type: Date,
+    required: true,
+  },
+  endDate: {
+    type: Date,
+    required: true
+  },
+  accepted: {
+    type: Boolean,
+    default: false,
+  },
+  declined: {
+    type: Boolean,
+    default: false,
+  },  
```

Comment:
I generally recommend to combine these into one property (e.g. `status`) and have an [enum validator](https://mongoosejs.com/docs/validation.html#built-in-validators) (e.g. `pending`, `approved`, `declined`, ...).

Reply:
I like this approach. This is actually a diff PR but I merged it to test the end points( https://github.com/hatchways/team-pumpernickel/pull/49 ). 

I was planning on having one prop and setting it as null when it's pending, since no action has been taken. I looked at forums to see if the behavior was fine. https://github.com/Automattic/mongoose/issues/6034#issuecomment-361822663.

I will implement this is the other PR. Thanks!